### PR TITLE
Enable weekly SOLVCON build schedule

### PR DESCRIPTION
**Pull request by tai271828**
Similar to what SOLVCON itself has done, let's enable a similar check for the build of the whole stack https://github.com/solvcon/solvcon/pull/248

### CODE REVIEW

File: .github/workflows/shunit2_runner.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: shunit2_runner
+name: Unittest - shunit2
```
Comment (tai271828) [inline .github/workflows/shunit2_runner.yml:1]: Simply for unifying naming convention

File: .github/workflows/solvcon_runner.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+name: Application Build - SOLVCON
+on:
+  schedule:
+    - cron: '21 18 * * 5'
```
Comment (tai271828) [inline .github/workflows/solvcon_runner.yml:4]: Run weekly. The content has been verified with my own Action run. Let's wait for the next Friday to see if the run is launched expectedly.

**Comment by tai271828** [comment]
@yungyuc if you have a moment, let's try to make this pull request land before Friday (so it can run its first scheduled job)?

**Review by yungyuc — APPROVED** [review]
LGTM.  Sorry for the late review.

**Pull request merged**